latest/Pipfile

```diff
@@ -34,6 +31,7 @@ pydrive = "*"
 #pyrax = "*"
 python-swiftclient = "*"
 requests_oauthlib = "*"
+httplib2 = "==0.15.0"
```

Comment:
Why pin to this exact version? If required then please comment in the code about why.

Reply:
It's been so long since I worked on this I don't recall. I ended up writing my own image instead and don't have time to work on it. Feel free to close the issue and MR.

Reply:
I've merged some of your changes, thanks for the PR though.